Exiled.Events/EventArgs/Player/EscapingPocketDimensionEventArgs.cs

```diff
@@ -47,5 +47,10 @@ public EscapingPocketDimensionEventArgs(Player player, Vector3 position)
         ///     Gets or sets a value indicating whether or not the player can successfully escape the pocket dimension.
         /// </summary>
         public bool IsAllowed { get; set; } = true;
+
+        //To Do
```

Comment:
TODO: also that comment shouldn't exist add a suggestion instead

Reply:
Removed